sfcasts/ep1/computed.md

```diff
@@ -1,228 +1,192 @@
-# Computed
+# Computed Properties
 
 Our collapsing sidebar works, but *yikes*, it looks terrible! We *need* to fix this!
-What we need to do here is hide all of these texts and links entirely when the sidebar
-is in its collapsed state. 
+Let's hide the text and links entirely when the sidebar
+is in its collapsed state.
 
 ## v-if directive
 
-Look at `sidebar.vue` in the template, we want to hide the `<h5>`, the `<ul>` and the
-`<hr />`, basically everything except for the button, when we're in a collapsed
-state.
```

Comment:
we had *just* said "when the sidebar is in its collapsed state" so this was unneeded. Less is more!